packages/manager/apps/pci-load-balancer/src/api/hook/useAddons/addons.select.ts

```diff
@@ -0,0 +1,8 @@
+import { TAddonRegions } from '@/types/region.type';
+
+export const selectHourlyAddons = (addons: TAddonRegions) =>
```

Comment:
to rename with filterByHourlyAddons or make a unit function isHourlyAddon that you can pass to your filter method